Gemfile

```diff
@@ -98,7 +98,7 @@ gem "savon",                          "~>2.2.0",   :require => false  # Automate
 gem "snmp",                           "~>1.2.0",   :require => false
 gem "uglifier",                       "~>2.7.1",   :require => false
 gem "sshkey",                         "~>1.8.0",   :require => false
-
+gem "byebug", "~> 8.2"
```

Comment:
Note, you can put this in Gemfile.dev.rb, in this same directory.  It's a untracked file for development gems you want to run locally but not force others to use.